### ISSUE TITLE
/music auto plays the music before the match is started.

### ISSUE THREAD

**Issue by zix3r** [yummypasta/MinecraftManhunt#16]
## Describe the bug
Setting the music to auto starts the music even before the game is started with the /start command and and sometimes it's not even the right one, like for example, I set the hunters and speedrunners, set the /music auto, and when I start it, no music 

## Potential Causes
I just did /music auto and it starts playing after a sec.

## Environment Info
**Plugin version:** 1.3

**Bukkit/Spigot server version:** 1.16.4

## Configuration file
Sending us your plugin `config.yml` file may help in diagnosing the issue. Paste the contents of the plugin's config file here: (feel free to omit your Discord token for security reasons)
```yaml
# How long the hunters should get blindness and slowness when the match starts.
headStartDuration: 15
compassEnabledInNether: false
sendUsageData: true
enableDiscord: true
discordToken: xxxxx
ip: verygood.ip
processDiscordCommands: false
discordServerId: 773923702651092992
musicChannelId: 775382857652633600
hunterRoleId: XXXXX
runnerRoleId: XXXXX
spectatorRoleId: XXXXX
uuid: 100c7e93-1313-4290-8941-aca8dea14701
```

## Server Logs
Please check your Server Console for any warnings or errors created by the plugin. If any are any, please copy and paste them here:
```
None.
```

## Anything else?
Nothing else.

**Comment by yummypasta** [comment]
Hello @zix3r,
This is most likely caused by your players doing things that are triggering special music events (like punching each other, etc). I'll see what I can do to stop music events from triggering before the game is started.

**Comment by zix3r** [comment]
Yep, they were punching each other, thanks for trying to fix this!!

**Comment by yummypasta** [comment]
I've fixed this issue in version 1.3.1! I'm not going to post the updated file on the Bukkit or Spigot websites since it's so minor, but you can download it on the [Releases tab](https://github.com/yummypasta/MinecraftManhunt/releases/tag/v1.3.1).
